frontend/src/Models/BarGraph.jsx

```diff
@@ -0,0 +1,22 @@
+import { BarChart } from "@mui/x-charts/BarChart";
+
+function BarGraph({ compareScores }) {
+  return (
+    <div className="barGraph">
+      <h2> Compare Playing Styles</h2>
+      <BarChart
+        dataset={compareScores}
+        xAxis={[{ scaleType: "band", dataKey: "style" }]}
+        yAxis={[{ scaleType: "linear", min: 0, max: 100 }]}
+        series={[
+          { dataKey: "myTeamScore", label: "My Team", stack: "A" },
+          { dataKey: "opponentScore", label: "Opponent", stack: "A" },
+        ]}
+        width={900}
+        height={400}
+      />
+    </div>
+  );
+}
```

Comment:
The chart looks great! but is a little hard to tell which team has higher scores from the stacked bars. Is there another chart type we can use?

Reply:
I was thinking the same thing but I was hesitant to make the bars next to each other because I think that the stacked balancing-out effect is a more fun visual. I was thinking of adding a horizontal line at the 50-50 mark to help the user tell where the middle point is. Do you think that would be more intuitive?